### ISSUE TITLE
Is this a code meaningless just for checking?

### ISSUE THREAD

**Issue by Green-16** [rasbt/mlxtend#707]
In the `mlxtend.evaluate` module ,the  `bias_variance_decomp` method has one line code that I don't know  the purpose .

`avg_expected_loss = (main_predictions != y_test).sum()/y_test.size
`
`avg_expected_loss = np.apply_along_axis(lambda x:
                                                (x != y_test).mean(),
                                                axis=1,
                                                arr=all_pred).mean() 
`
The variable avg_expected_loss  is assigned twice. Is this just to check the value?

**Comment by rasbt** [comment]
Thanks for the note, the line

    avg_expected_loss = (main_predictions != y_test).sum()/y_test.size 

doesn't do anything, indeed, and shouldn't be there. Will remove.

**Comment by Green-16** [comment]
ok.Thank you for replying me.
